Prace_domowe/Praca_domowa1/Grupa2/Karol_Degorski/pd2.ipynb

```diff
@@ -0,0 +1,1209 @@
+{
```

Comment:
<p>Prawda, choć w tym miejscu należy rozróżnić transformację na potrzeby EDA i transformację na potrzeby regresji – do EDA wystarczy przerobić dni na 1-7 i miesiące na 1-12 (co nie zawsze jest konieczne, ale bywa wygodne), do regresji wypadałoby to zamienić na one-hot encoding.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>